src/Avalonia.Controls/Primitives/Track.cs

```diff
@@ -154,7 +154,11 @@ protected override Size ArrangeOverride(Size finalSize)
 
                 if (increaseButton != null)
                 {
-                    increaseButton.Arrange(new Rect(firstWidth + thumbWidth, 0, remaining - firstWidth, finalSize.Height));
+                    increaseButton.Arrange(new Rect(
```

Comment:
Should probably apply the same logic to the Vertical case (line 192).

Reply:
The vertical case on line 192 already seems to have this logic. Am I missing something?

Reply:
Hmm, it's not exactly the same. There's no Max on finalSize.Width. BTW I'm almost sure this shouldn't happen. Shouldn't ArrangeOverride always receive a valid size?

I'd also format the vertical and horizontal the same way.

Reply:
`ArrangeOverride` _does_ always receive a valid size, this problem is caused by a floating point rounding error where at the line of the fix in the added unit test, we have the follow values:

```
remaining = 120.12689393939394
firstWidth = 120.12689393939395
```

So `finalSize.Width ` doesn't need a `Max` because there's no `-` going on there and so no potential for a negative value caused by an FP rounding error.

Reply:
Updated the formatting btw.

Reply:
So you can also change `Math.Max(0, finalSize.Height)` to simply `finalSize.Height`, right?

Reply:
Ah, yes I can!

Reply:
@aelij fixed!